src/app/pages/MainArticles/components/MainArticles/MainArticles.component.spec.jsx

```diff
@@ -0,0 +1,48 @@
+import renderer from 'react-test-renderer';
+import React from 'react';
+import { compose } from 'redux';
+import { shallow, mount } from 'enzyme';
+
+import {
+  withProviders,
+  withAdditionalProperties,
+} from '@core/tests/theme-helpers';
+
+import MainArticles from './MainArticles.component';
+
+const fetchArticlesMock = jest.fn().mockReturnValue([
```

Comment:
Add more tests here, mock list of items which should be loaded here if possible, and check if spoecific number fo items is loaded into DOM.